src/pages/catalogC/catalogC.js

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import connect from 'react-redux/es/connect/connect';
+
+import Modal from 'patternfly-react/dist/esm/components/Modal/Modal';
+import EmptyState from 'patternfly-react/dist/esm/components/EmptyState/EmptyState';
+import { Alert } from 'patternfly-react/dist/esm/components/Alert';
+
+import CatalogView from '../../components/catalogView';
+import { fetchCatalogItems } from '../../redux/actions/catalogActions';
+import { helpers } from '../../common/helpers';
+
+class CatalogC extends React.Component {
```

Comment:
please use a meaningful name for this class.
CatalogC is not so meaningful
also rename to not use camelCase and add copyright headers